cmd/s3_to_redshift.go

```diff
@@ -69,12 +70,8 @@ func runCopy(db *redshift.Redshift, inputConf s3filepath.S3File, inputTable reds
 	}
 
 	// COPY direct into it, ok to do since we're in a transaction
-	// assuming that we always want to copy from s3 and have gzip, so last 2 params are true
+	// assuming that we always want to copy from s3, so that parameter is always true
```

Comment:
Nit. Not actually a sentence, so this took me a while to parse.
I'm also not seeing any s3 parameters, so is this even an accurate comment, still?